### ISSUE TITLE
Does it work on iOS simulator?

### ISSUE THREAD

**Issue by wan54** [NativeScript/sample-PhotoAlbum#1]
The count of "Total images" increases as I click on the button but nothing in the list?

**Comment by nikolay-diyanov** [comment]
Yes, it works, but the final state of the application that you can find at NativeScript/sample-PhotoAlbum assumes that you have a Telerik Backend Services project containing images and that you have set the appropriate API Key replacing the message "YOUR API KEY". Regarding the button message, it refers to the count of an array of locally stored images. If you want to see the images without creating a Telerik Backend Services project, just make the "photoItems" property return "array" rather than "backendArray". Let me know if that has helped or if you need additional assistance.
